testing/citest/tests/google_server_group_test.py

```diff
@@ -161,6 +164,7 @@ def create_server_group(self):
       'instanceMetadata': {
         'load-balancer-names': self.__lb_name
       },
+      'userData': 'customUserData=' + self.__custom_user_data,
```

Comment:
this doesnt look right. If it is then 'customUserData' should be parameterized as well. I didnt realize this was. (__test_custom_user_data_key, __test_custom_user_data_value)